lib/events/joinrulesevent.cpp

```diff
@@ -0,0 +1,16 @@
+// SPDX-FileCopyrightText: 2019 Kitsune Ral <Kitsune-Ral@users.sf.net>
```

Comment:
I didn't remember writing this code :) Can you link to the exact place you picked it from?

Reply:
> I didn't remember writing this code :) Can you link to the exact place you picked it from?

https://invent.kde.org/network/neochat/-/blob/master/src/joinrulesevent.cpp
copyed from neochat >_<

Reply:
@CarlSchwan, Git says it's yours, rather? :)
```suggestion
// SPDX-FileCopyrightText: 2022 Carl Schwan <carl@carlschwan.eu>
```

Reply:
yeah looks like I made a mistake when creating the file and put the wrong name in the header